config/environment.js

```diff
@@ -31,6 +31,7 @@ module.exports = function(environment) {
     // ENV.APP.LOG_VIEW_LOOKUPS = true;
     ENV.APP.API_HOST = 'http://127.0.0.1:8000';
     ENV.APP.API_NAMESPACE = 'api';
+    ENV.APP.JOB_WATCHER_HOST = 'localhost:8080';
```

Comment:
Somewhat inconsistent that the `API_HOST` includes the protocol, but the `JOB_WATCHER_HOST` does not.